### DISCUSSION TITLE
Welcome to TWRP-device-tree-generator Discussions!

### DISCUSSION THREAD

**Discussion by SebaUbuntu** [SebaUbuntu/TWRP-device-tree-generator#26] [General]
## 👋 Welcome!
  You can use this space to ask questions, tell some ideas, show something, etc. without opening issues

**Comment by vinidossantos** [comment]
Test :3

**Comment by SakthivelNadar** [comment]
OwO

**Comment by portsample** [comment]
Ciao mondo!

**Comment by rabba83** [comment]
Ciao Seba, scusa ma non ho capito cosa intendi con Open a terminal pointing to the root of the script. io ho provato ad eseguire il comando pip3 install "indirizzo cartella twpr-device-tree-generator" ed mi ha dato errore in rosso

**Reply by SebaUbuntu** [reply]
> Ciao Seba, scusa ma non ho capito cosa intendi con Open a terminal pointing to the root of the script. io ho provato ad eseguire il comando pip3 install "indirizzo cartella twpr-device-tree-generator" ed mi ha dato errore in rosso

Apri un nuovo thread nella sezione "Issues", possibilmente in inglese

**Reply by rabba83** [reply]
No fa lo stesso, intanto non ci riesco. Grazie

**Comment by duarteoficialhp** [comment]
# ~~****~~
